src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Cognitive Services ComputerVision SDK")]
 [assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services ComputerVision APIs.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
The `AssemblyVersion` stays the same, `AssemblyFileVersion` changes

Reply:
Ahh sorry, missed this.